pkg/apis/camel/v1alpha1/integrationplatform_types.go

```diff
@@ -105,6 +105,7 @@ type IntegrationPlatformBuildSpec struct {
 	PersistentVolumeClaim string                                  `json:"persistentVolumeClaim,omitempty"`
 	Maven                 MavenSpec                               `json:"maven,omitempty"`
 	HTTPProxySecret       string                                  `json:"httpProxySecret,omitempty"`
+	Cache                 bool                                    `json:"cache,omitempty"`
```

Comment:
Could you please name the API field more specifically as well.

Reply:
Thanks @astefanutti for the suggestion, I have updated it now